submissions/tests/test_team_api.py

```diff
@@ -195,23 +195,42 @@ def test_create_submission_for_team_existing_individual_submission(self):
         """
         Test for calling create_submission_for_team when a user somehow already has
         an existing active submission for the item.
-
-        For normal Submissions, if a submission exists and we create another one, the second
-        will increment the first's attempt_number. However, for team submissions, we currently
-        pass the attempt_number from team_api.create_submission to api.create_submission, so
-        all created individual submissions will have the same attempt_number
+        This can happen if a user was on a team that submitted and then is reassigned to a team
+        that didn't have a submission and is submitting.
+        Expected outcome: submission is created because the previous submission was not team based
         """
         user_3_item = self._get_or_create_student_item(self.anonymous_user_id_map[self.user_3])
         SubmissionFactory.create(student_item=user_3_item)
         team_submission_data = self._call_create_submission_for_team_with_default_args()
         self.assertEqual(team_submission_data['attempt_number'], 1)
-        submissions = Submission.objects.select_related(
+        submission_student_ids = [sub.student_item.student_id for sub in Submission.objects.select_related(
             'student_item'
-        ).filter(
-            uuid__in=team_submission_data['submission_uuids']
-        ).all()
-        for submission in submissions:
-            self.assertEqual(submission.attempt_number, 1)
+        ).filter(uuid__in=team_submission_data['submission_uuids']).all()]
+        self.assertIn(user_3_item.student_id, submission_student_ids)
+
+    def test_create_submission_for_team_after_reassignment(self):
```

Comment:
This is a kinda odd case, just since it's not expected behavior, as in we shouldn't really expect a case where every member of a team already has a submission. If anything, this should be an error state where we don't create the TeamSubmission and raise an exception